<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMapRoute</class>
 <widget class="QWidget" name="frmMapRoute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMap" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string>原始数据</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidgetSource"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>筛选数据</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidgetTarget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labTip">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>角度 0°/第 0 个/共 0 个</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="ckSelectAddr">
             <property name="text">
              <string>地图选点</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbtnStartAddr">
             <property name="text">
              <string>起点</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbtnEndAddr">
             <property name="text">
              <string>终点</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labStartAddr">
           <property name="text">
            <string>起点地址</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtStartAddr">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labEndAddr">
           <property name="text">
            <string>终点地址</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtEndAddr">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labRouteType">
           <property name="text">
            <string>路线方式</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cboxRouteType">
           <item>
            <property name="text">
             <string>公交</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>驾车</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>步行</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>骑行</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labPolicyType">
           <property name="text">
            <string>路线方案</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cboxPolicyType"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labPointCount">
           <property name="text">
            <string>目标点数</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtPointCount">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labDeviceName">
           <property name="text">
            <string>设备名称</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="txtDeviceName">
           <property name="text">
            <string>国产大飞机C919</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labMoveInterval">
           <property name="text">
            <string>移动间隔</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="cboxMoveInterval"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labMoveMode">
           <property name="text">
            <string>轨迹模式</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="cboxMoveMode"/>
         </item>
         <item row="9" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnCheckData">
             <property name="text">
              <string>筛选数据</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnTestData">
             <property name="text">
              <string>模拟轨迹</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btnSearchData">
             <property name="text">
              <string>查询路线</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnDrawData">
             <property name="text">
              <string>绘制路线</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ckSelectAddr</tabstop>
  <tabstop>rbtnStartAddr</tabstop>
  <tabstop>rbtnEndAddr</tabstop>
  <tabstop>txtStartAddr</tabstop>
  <tabstop>txtEndAddr</tabstop>
  <tabstop>cboxRouteType</tabstop>
  <tabstop>cboxPolicyType</tabstop>
  <tabstop>txtPointCount</tabstop>
  <tabstop>txtDeviceName</tabstop>
  <tabstop>cboxMoveInterval</tabstop>
  <tabstop>cboxMoveMode</tabstop>
  <tabstop>btnSearchData</tabstop>
  <tabstop>btnTestData</tabstop>
  <tabstop>btnCheckData</tabstop>
  <tabstop>btnDrawData</tabstop>
  <tabstop>tableWidgetSource</tabstop>
  <tabstop>tableWidgetTarget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
